notebooks/CustomerScenarios/Case Study - Using EconML to evaluate the treatment effect of training program - Lalonde dataset.ipynb

```diff
@@ -0,0 +1,1184 @@
+    "                         basic_ols_controls=basic_ols_columns,\n",
+    "                         complete_ols_controls=complete_ols_columns_female,\n",
+    "                         econml_controls=econml_controls_female,\n",
+    "                         outcome_name=outcome_name_female,cols_to_scale=7\n",
+    "                         )"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Numbers under each method represent [point stimate, standard error]. \n"
+     ]
+    },
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>control_name</th>\n",
+       "      <th># of obs</th>\n",
+       "      <th>earning_growth</th>\n",
+       "      <th>OLS</th>\n",
+       "      <th>OLS full controls</th>\n",
+       "      <th>DML full controls</th>\n",
+       "      <th>DR full controls</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>exp controls</td>\n",
+       "      <td>585</td>\n",
+       "      <td>2954</td>\n",
+       "      <td>[858, 307]</td>\n",
+       "      <td>[880, 307]</td>\n",
+       "      <td>[831, 307]</td>\n",
+       "      <td>[833, 307]</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>psid1</td>\n",
+       "      <td>648</td>\n",
+       "      <td>1316</td>\n",
+       "      <td>[-2730, 441]</td>\n",
+       "      <td>[1068, 529]</td>\n",
+       "      <td>[1155, 701]</td>\n",
+       "      <td>[604, 631]</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>psid2</td>\n",
+       "      <td>182</td>\n",
+       "      <td>2412</td>\n",
+       "      <td>[-90, 514]</td>\n",
+       "      <td>[510, 560]</td>\n",
+       "      <td>[880, 717]</td>\n",
+       "      <td>[827, 806]</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   control_name  # of obs  earning_growth           OLS OLS full controls  \\\n",
+       "0  exp controls       585            2954    [858, 307]        [880, 307]   \n",
```

Comment:
Is it just a coincidence that all 4 methods have a standard error of exactly 307?

Reply:
they are all equal to 307 after round to integer.